.github/workflows/build_test.yml

```diff
@@ -230,6 +230,7 @@ jobs:
         env:
           PYTHON_VERSION: ${{ matrix.python-version }}
           cloud_provider: ${{ matrix.cloud-provider }}
+          PYTEST_ADDOPTS: --color=yes
```

Comment:
Nice.

Reply:
You can see this in action on the Checks tab. 😄